.github/workflows/build-mac-arm64.yml

```diff
@@ -1,45 +0,0 @@
-name: Build mac arm64
-
-on:
-  workflow_dispatch:
-
-env:
-  GITHUB_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
-  GH_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
-  RUDDERSTACK_CLIENT_ID: ${{ secrets.RUDDERSTACK_CLIENT_ID }}
-
-jobs:
-  release-mac-arm64:
-    runs-on: macos-latest-xlarge
```

Comment:
we need this m1 runner because we compile to bytecode and the bytecode has to be compiled on the same arch that it will be distributed for https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/pull/574